cardano-node-chairman/cardano-node-chairman.cabal

```diff
@@ -51,7 +51,8 @@ executable cardano-node-chairman
                       , io-classes ^>= 0.3
                       , optparse-applicative-fork
                       , ouroboros-consensus
-                      , ouroboros-network
+                      , ouroboros-network-api
+                      , ouroboros-network-protocols
```

Comment:
These should have bounds, unless there is a bound on `cardano-node`.

Reply:
This is a problem that inflicts lots of packages.  I'm intending to add them in a separate PR and fix it up for all packages.